.gitingore

```diff
@@ -0,0 +1,11 @@
+dist/
+.vscode
+.idea
+node_modules/
+npm-debug.log
+example/keys/*.pem
+.DS_Store
+**/.DS_Store
+coverage
+.nyc_output
+coverage.lcov
```

Comment:
Nitpick: A lot of these files aren't here at all (such as `dist, .vscode, .idea`).

Reply:
copy paste from EG